asr_automation/asr_automation_pipeline.ipynb

```diff
@@ -0,0 +1,347 @@
+      ],
+      "metadata": {
+        "id": "FxLPYZEb0sdL"
+      },
+      "id": "FxLPYZEb0sdL"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "5010f2df",
+      "metadata": {
+        "id": "5010f2df"
+      },
+      "outputs": [],
+      "source": [
+        "! pip install superannotate #SA SDK installation\n",
+        "! pip install git+https://github.com/openai/whisper.git #install OpenAI Whisper\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 1.1  User Variables Setup"
+      ],
+      "metadata": {
+        "id": "-znAjTBVotIT"
+      },
+      "id": "-znAjTBVotIT"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "SuperAnnotate Python SDK functions work within the team scope of the platform, so a team-level authorization is required. To authorize the package in a given team scope, get the authorization token from the team settings page."
+      ],
+      "metadata": {
+        "id": "rliZ7tO3rgb5"
+      },
+      "id": "rliZ7tO3rgb5"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "SDK_TOKEN = \"ADD_YOUR_TOKEN_HERE\" #Your SuperAnnotate SDK token\n",
+        "PROJECT_NAME = \"ADD_YOUR_PROJECT_NAME_HERE\" #A project with this name will be created to upload all voice files and transcribtions"
+      ],
+      "metadata": {
+        "id": "NXVjtPPElHEh"
+      },
+      "id": "NXVjtPPElHEh",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#1.2 Constants Setup"
+      ],
+      "metadata": {
+        "id": "ucDzc9DLo4_a"
+      },
+      "id": "ucDzc9DLo4_a"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "S3_BASE_URL = \"https://sa-public-datasets.s3.amazonaws.com/audio-datasets/us-elections-2020-debates/\"\n",
+        "DATASET_FILES = [\n",
+        "    \"us_election_2020_1st_presidential_debate.mp3\",\n",
+        "    \"us_election_2020_2nd_presidential_debate.mp3\",\n",
+        "    \"us_election_2020_biden_town_hall.mp3\",\n",
+        "    \"us_election_2020_trump_town_hall.mp3\",\n",
+        "    \"us_election_2020_vice_presidential_debate.mp3\"\n",
+        "]"
+      ],
+      "metadata": {
+        "id": "Lp0ODWJBlqTT"
+      },
+      "id": "Lp0ODWJBlqTT",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#1.3 Download the data"
+      ],
+      "metadata": {
+        "id": "U9n7c0sBrnE3"
+      },
+      "id": "U9n7c0sBrnE3"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from os.path import join\n",
+        "import requests\n",
+        "import os\n",
+        "\n",
+        "os.makedirs('data', exist_ok=True)\n",
+        "for dataset_file in DATASET_FILES:\n",
+        "  response = requests.get(join(S3_BASE_URL, dataset_file))\n",
```

Comment:
Is it the only one way to do so? If not, so maybe we need to talk with Sergey, who use S3 bucket too, and fix this step for every tutorial ?